admin-dev/themes/new-theme/scss/pages/_product_page.scss

```diff
@@ -1065,3 +1065,63 @@
     }
   }
 }
+
+#select2-product_combination_bulk_images-results .select2-results__option--highlighted[aria-selected],
+#select2-product_combination_bulk_images-results .select2-results__option[aria-selected=true] {
+  background-color: inherit;
+  color: inherit;
+}
+
+.combination-bulk-images {
+  width: 2.857rem;
+  height: 2.857rem;
+  background-color: $gray-light;
+  border: solid 0.07143rem $gray-light;
+}
+
+.combination-bulk-images-label {
+  height: 1.429rem;
+  object-fit: contain;
+  font-family: $font-family-base;
+  font-size: 0.9rem;
+  line-height: 1.43;
+  color: $gray-medium;
+  margin: 0.6429rem 0 0 0.6429rem;
+}
+
+#select2-product_combination_bulk_images-results {
+  li.select2-results__option input[type=checkbox] {
+    margin: 0 0.625rem 0 0.313rem;
+    vertical-align: middle;
+  }
+
+  &::-webkit-scrollbar {
+    width: 0.5714rem;
+    background-color: rgba(187, 205, 210, 0.3);
+  }
+
+  &::-webkit-scrollbar-thumb {
+    width: 0.5714rem;
+    border-radius: 7.143rem;
+    background-color: $gray-light;
+  }
+
+}
+
+#select2-product_combination_bulk_images-container .select2-container--prestakit .select2-selection--multiple {
+  padding: inherit;
+  padding-right: 1.429rem;
+
+  &::before {
+    content: "\E313";
+    font-family: Material Icons;
+    position: absolute;
+    top: 50%;
+    right: 0.3571rem;
+    transform: translateY(-50%);
+  }
+
+  ul {
+    display: none;
+  }
+}
```

Comment:
It seems the products label are missing:
![capture d ecran 2017-09-25 a 10 44 03](https://user-images.githubusercontent.com/206733/30800136-50de9382-a1df-11e7-9b56-e2c691ed5cab.png)